src/topology/topological_fiber_bundle.lean

```diff
@@ -781,8 +793,6 @@ structure topological_fiber_bundle_core (ι : Type*) (B : Type*) [topological_sp
 (coord_change_comp : ∀i j k, ∀x ∈ (base_set i) ∩ (base_set j) ∩ (base_set k), ∀v,
   (coord_change j k x) (coord_change i j x v) = coord_change i k x v)
 
-attribute [simp, mfld_simps] topological_fiber_bundle_core.mem_base_set_at
```

Comment:
Because one of the other two choices that makes the `simp` work smoothly is to decide if we want to use `Z.base_set i` or `(Z.local_triv_ext i).base_set` and I think that clearly the preference should go to the second alternative, so instead of declaring simp `topological_fiber_bundle_core.mem_base_set_at` I declare simp `lemma mem_local_triv_at_ext_base_set`.

Do you want me to justify my choice for giving preference to `(Z.local_triv_ext i).base_set` over `Z.base_set i` or do you agree with this choice?

Reply:
I don't understand. The simp attribute ensures that, when `simp` sees a goal `x ∈ Z.base_set (index_at x)`, then it solves it automatically. The other lemmas ensure that it solves automatically other goals. So, all these simp attributes are compatible, and we can have all of them together. Am I overlooking something here?

Reply:
The current `simp` works like this: `x ∈ Z.base_set (index_at x)` is simplified to `x ∈ (Z.local_triv_at_ext x).base_set` and then this is solved automatically! So your `simp` is replaced by two other simps. All of the 3 cannot be simp otherwise Lint complains.